Userland/Libraries/LibJS/Tests/classes/class-methods.js

```diff
@@ -49,3 +49,15 @@ test("extended name syntax", () => {
     expect(a[12]()).toBe(2);
     expect(a.hello()).toBe(3);
 });
+
+test("method named async", () => {
```

Comment:
```suggestion
test("method named 'async'", () => {
```

just for consistency with all the others :)